graylog2-server/src/main/java/org/graylog2/rest/models/system/contenpacks/responses/ContentPackMetaData.java

```diff
@@ -0,0 +1,23 @@
+package org.graylog2.rest.models.system.contenpacks.responses;
```

Comment:
This file is missing the license header and that makes the build fail. You can fix that by running `mvn license:format` in the server repository.